cli/tests/utils/mod.rs

```diff
@@ -0,0 +1,211 @@
+use std::time::Duration;
+
+use tokio::time::sleep;
+use zingo_cli::regtest::RegtestManager;
+use zingolib::lightclient::LightClient;
+
+pub async fn increase_height_and_sync_client(
+    manager: &RegtestManager,
+    client: &LightClient,
+    n: u32,
+) {
+    let start_height = client
+        .do_wallet_last_scanned_height()
```

Comment:
If the client (or lightwalletd) isn't fully synced when this is called, then `start_height` will lead to an improperly determined target, which I think will depending on the numbers involved either early-return or hang forever.

Reply:
I believe this is fixed by running `do_sync` immediately before this line.

Reply:
Though..  this fix assumes that `do_sync` fulfills its contract reliably.